pyartcd/pyartcd/locks.py

```diff
@@ -11,7 +11,7 @@ class Lock(enum.Enum):
     OLM_BUNDLE = 'olm-bundle-lock-{version}'
     MIRRORING_RPMS = 'mirroring-rpms-{version}'
     COMPOSE = 'compose-lock-{version}'
-    GITHUB_ACTIVITY = 'github-activity-lock-{version}'
+    DISTGIT_REBASE = 'distgit-rebase-lock-{version}'
```

Comment:
Nitpick, but this can just be `rebase-lock` and distgit would be implied?

Reply:
I wanted to make it clear that it's Distgit that we're protecting. Rebase can happen locally and does not necessarily imply interaction with distgit